study-builder/studies/rarex/consent-lar-assent.conf

```diff
@@ -665,7 +673,187 @@
           },
           "blockType": "QUESTION",
           "shownExpr": null
-        }//,
+        },
+        {
+          "titleTemplate": {
+            "templateType": "HTML",
+            "templateText": "<h4 class="subtitle subtitle--light">$consent_witness_signature_title</h4>",
```

Comment:
Can you remove all of these new "witness" blocks as well? No need to have these around. Can you remove them from the other files as well?

Reply:
working on that